src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,10 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie", Movie.class).getResultList();
```

Comment:
It is better to write all SQL commands in upper case, fix them eveywhere
```suggestion
            return session.createQuery("FROM Movie", Movie.class).getResultList();
```